engine/src/test/java/net/jqwik/api/ListArbitraryTests.java

```diff
@@ -97,7 +97,7 @@ void uniqueElementsWithNull(@ForAll Random random) {
 		ListArbitrary<Integer> listArbitrary =
 			Arbitraries.integers().between(1, 1000).injectNull(0.5)
 					   .list().ofMaxSize(20)
-					   .uniqueElements(i -> i % 100);
+					   .uniqueElements(i -> i == null ? null : i % 100);
```

Comment:
`i` is `Integer i`.
Since the generator has `injectNull`, I believe it is reasonable that the user must handle `null` case, and they must not attempt to unbox `null` value.

Reply:
`i` is `Integer` because `int` cannot be used as a type parameter.

I'd like to keep the above behaviour unchanged.

Reply:
Well. The issue with jqwik is that it hides an exception from a userprovided code.
In other words, consider `.uniqueElements(i -> computeNameOf(i));`
If `computeNameOf` throws NPE, then jqwik would hide that, and the users won't detect the error.

I believe it is absolutely unexpected and unexpected behavior.

Reply:
Let me think about that for a while.